ansible/group_vars/all/main.yml

```diff
@@ -0,0 +1,5 @@
+repos_dir: /opt/pantheon-repos
+kratos_dir: "{{ repos_dir }}/kratos"
+moirai_dir: "{{ repos_dir }}/moirai"
+dash_dir: "{{ repos_dir }}/dash"
+data_dir: /data
```

Comment:
Wouldn't it make sense to have all the files for the project in one location? example:

```
/opt/pantheon/repos/
/opt/pantheon/data/
```